ERCS/erc-7779.md

```diff
@@ -0,0 +1,191 @@
+---
+eip: 7779
+title: Interoperable Delegated Accounts
+description: Interface for delegated externally owned accounts to enable better redelegation between wallets.
+author: David Kim (@PowerStream3604), Richard Meissner (@rmeissner), Akshay Patel (@akshay-ap), Joshua Kim (@LightningHun), Fangting (@trinity-0111), Yoav Weiss (@yoavw)
+discussions-to: https://ethereum-magicians.org/t/erc-7779-interoperable-delegated-account/21237
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-10-02
+requires: 7201, 7702
+---
+
+## Abstract
+
+This proposal outlines the interfaces to make delegated EOAs interoperable after the merge of [EIP-7702](./eip-7702.md). With [EIP-7702](./eip-7702.md), EOAs will be able to enable execution abstraction, which leads to a more feature-rich account, including gas sponsorship, batch execution, and more.
+
+However, there is a need to help facilitate storage management for redelegation, as invalid management of storage may incur storage collisions that can lead to unexpected behavior of accounts (e.g., account getting locked, security vulnerabilities, etc)
+
+The interface `InteroperableDelegatedAccount` suggests the interfaces for delegated EOAs to be interoperable and facilitate a better environment for redelegation.
```

Comment:
> The interface `InteroperableDelegatedAccount` suggests the interfaces

I would change the second occurrence of 'interface' with 'methods' or 'functions'